### PR TITLE
[W5.7][F10-3] Lim Wai Lun

### PR DESCRIPTION
**Pull request by wailu**
Add a welcome feature to welcome new users

### CODE REVIEW

**Comment by dongsiji** [comment]
very good job, maybe can include some testing of your welcome next time

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -73,33 +63,36 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case AddCommand.COMMAND_WORD:
+                return prepareAdd(arguments);
```
Comment (liaujianjie) [inline src/seedu/addressbook/parser/Parser.java:67]: Remember to follow the repo's coding conventions and style guides! You can change the indentation rules for switch-cases in your IDE settings.

**Review by liaujianjie — APPROVED** [review]
Only one small comment

**Review by eehooi — COMMENTED** [review]
Would be good to update the User Guide as well

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (dalessr) [inline src/seedu/addressbook/parser/Parser.java:14]: Coding standard violation here.

**Review by dalessr — APPROVED** [review]
@wailunlim Well done for the `welcome` command 👍 Remember to update user guide & add JUnit test for the new feature.

Please close the PR after looking at the comments below.

**Pull request closed without merging**